### PR TITLE
Enviroment var to expand test cases

### PR DESCRIPTION
**Pull request by prury**
Added a fixed mnemonic inside a Github Secret variable to expand test cases.
Before, we had a random mnemonic without balance or nfts in it, making it impossible to test confirmation modals, mass minting, shopping cart and masslisting. 

Tests are not probably not passing because of this:
image
or this: 
image

But as you can see, a copy of this PR on my forked repository with the same secret set works just fine: 
[Prury nft-gallery](https://github.com/prury/nft-gallery/pull/1)

## PR Type

- [x] Testing feature

## Context

- [ ] Related to: #7826
- [ ] Requires deployment <snek/rubick/worker>

### <samp>🤖 Generated by Copilot at 568beaf</samp>

This pull request enables the use of a secret mnemonic phrase for end-to-end testing, and updates the e2e-login page and the test cases accordingly. It also fixes a minor bug in the createcollection test.

### <samp>🤖 Generated by Copilot at 568beaf</samp>

> _We use the secret phrase to enter the realm_
> _We test the account with the avatar's helm_
> _We create new collections with our funds_
> _We fix the broken tests with our guns_

### CODE REVIEW

**Comment by exezbcz** [comment]
I thought this problem was due to our disabling carbonless nfts by default lmao

**Comment by prury** [comment]
> I thought this issue was due to our disabling carbonless nfts by default lmao

Hehe, I'm trying to do that account thing we we're talking yesterday, but no success yet

File: .github/workflows/playwright.yml [hunk 1]
```diff
@@ -13,6 +13,11 @@ jobs:
     - uses: actions/setup-node@v3
       with:
         node-version: 18
+    
+    - name: Set Enviroment Variable
+      run: |
+        touch .env
+        echo VUE_APP_MNEMONIC_E2E=${{ secrets.MNEMONIC_E2E }} >> .env
```
Comment (roiLeo) [inline .github/workflows/playwright.yml:20]: No need to prefix with `VUE_APP_*` that's for Vue2

**Review by roiLeo — COMMENTED** [review]
Please update `.env.example` with this new env var

**Comment by prury** [comment]
> Please update `.env.example` with this new env var

done

**Review by roiLeo — APPROVED** [review]

**Review by preschian — APPROVED** [review]

**Comment by prury** [comment]
@yangwao merge?

**Review by Jarsen136 — APPROVED** [review]

**Comment by roiLeo** [comment]
👀 
Screenshot 2023-11-02 at 15-02-03 Enviroment var to expand test cases by prury · Pull Request #7849 · kodadot_nft-gallery

**Comment by prury** [comment]
> 👀 Screenshot 2023-11-02 at 15-02-03 Enviroment var to expand test cases by prury · Pull Request #7849 · kodadot_nft-gallery

it is supposed to fail here @roiLeo, since PR from a forked repo cannot access the secret environment var

**Comment by roiLeo** [comment]
> it is supposed to fail here @roiLeo, since PR from a forked repo cannot access the secret environment var you can observe that all the fails are related to e2e login

What should the `MNEMONIC_E2E` be like? is it supposed to be your mnemoni key from your account?

**Comment by prury** [comment]
> > it is supposed to fail here @roiLeo, since PR from a forked repo cannot access the secret environment var you can observe that all the fails are related to e2e login
> 
> What should the `MNEMONIC_E2E` be like? is it supposed to be your mnemoni key from your account?

hahaha, ofc not! its a fresh mnemonic with small amount of currency for testing purposes

**Comment by yangwao** [comment]
@prury is there anything we should add/change before merging or we can just merge it?

**Comment by prury** [comment]
> @prury is there anything we should add/change before merging or we can just merge it?

I've been discussing some other options with @roiLeo, better to wait a bit more for now

**Comment by roiLeo** [comment]
will it blend?

**Pull request merged**